docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
       - internal
     depends_on:
       - ldap
+    links:
+      - useradm:ldap
```

Comment:
links is from version 1 and was replaced with depend_on in v2 and is better as it handles both parts of the linl and makes sure the containers are started in the right order